File: 프로그래머스/lv0/120816. 피자 나눠 먹기 （3）/피자 나눠 먹기 （3）.py
def solution(slice, n):
    answer = 0
    
    a = n / slice
    b = n % slice
    
    if b == 0:
        answer = a * 1
    else:
        answer = a * 1 + 1
        
    answer = int(answer)
    print(answer)
    
    return answer

File: 프로그래머스/lv0/120816. 피자 나눠 먹기 （3）/피자 나눠 먹기 （3）.js
function solution(slice, n) {
    var answer = 0;
    
    a = Math.floor(n / slice)
    b = n % slice
    
    if (b === 0) {
        answer = a * 1;
    } else {
        answer = a * 1 + 1;
    }
    
    return answer;
}